x-pack/plugins/cases/public/components/all_cases/table.tsx

```diff
@@ -61,12 +61,6 @@ const BasicTable = styled(EuiBasicTable)`
     &.isSelectorView .euiTableRow.euiTableRow-isExpandedRow:hover {
       background-color: transparent;
     }
-
```

Comment:
@cnasikas is there other stuff I need to delete here? Maybe the `isSelectable`?

Reply:
I think we can remove the `isSelectable`. I tested the modal and it is not using it.

Reply:
Looks like this is what we had previously:

https://github.com/elastic/kibana/commit/a5e0cce12e615d23fe69a3f6bfcea5eba192f9da#diff-e67bc6aa11f1633b542641f642933f1cef7b9cdb32510ee8365f734f30b12f2cL89

Should we leave this one there or remove the entire styled component?

```
.euiTableRow-isExpandedRow.euiTableRow-isSelectable .euiTableCellContent {
    padding: 8px 0 8px 32px;
  }
```

Reply:
I think we should remove everything because the CSS rules are about either `isExpandedRow` or `isDisabled` which was for subcases.